docs/src/00-doc/03-create-reusable-components/02_designing_components.stories.mdx

```diff
@@ -222,23 +237,23 @@ Many of the handover items should already be covered if the design process has b
   - **Visual** specification (sizes, paddings, typographic styles, colors) are fully defined using design tokens ([See 2.2](#22-using-tokens-to-specify-a-new-component)).
   - **States**’ specifications (e.g. default, hover, active, focus, error, disabled) and **interactive behavior** are defined and fully specified. In both cases, the user actions leading to each state should also be indicated.
   - **Responsive** behavior specifications are also outlined <br/>
-<br/>
+    <br />
 
 👉🏻 The system designer will be in charge of creating and making available, before development, the component-level token files based on the design specifications.
 
 - Basic component documentation (as defined in 1.4):<br/>
   - A distinctive **name** for the component and its relevant typologies
   - A **short description** of the overall purpose of the component<br/>
   - **Accessibility** specifications, such as keyboard navigation, are defined<br/>
-  <br/>
+    <br />
 - Storybook indications:<br/>
   - Specify where, within the current **Storybook structure**, the component should be classified (new dedicated section or subsection), based on the relationship to its component family (this relationship should also be expressed by the name).
   - Indicate the required Storybook **controls**, if needed: these allow users to modify some properties of a component in real time.<br/>
-  <br/>
+    <br />
```

Comment:
and here

Reply:
That was definitely not on purpose 🤔  Fixing